be/src/Project/Services/Client/Client.Api/Infrastructure/Clients/Entities/Address.cs

```diff
@@ -6,12 +6,6 @@ namespace Client.Api.Infrastructure.Clients.Entities
 {
     public class Address : Entity
     {
-        public string Company { get; set; }
```

Comment:
**suggestion:** let's keep these fields because in the address can contain a name of the company or recipient first name / last name

Reply:
Okay, but what you think to set the property name as Recipient?